controllers/imagerepository_controller.go

```diff
@@ -48,10 +48,11 @@ const (
 
 	ImageRepositoryFinalizer = "appstudio.openshift.io/image-repository"
 
-	buildPipelineServiceAccountName = "appstudio-pipeline"
-	updateComponentAnnotationName   = "image-controller.appstudio.redhat.com/update-component-image"
-	additionalUsersConfigMapName    = "image-controller-additional-users"
-	additionalUsersConfigMapKey     = "quay.io"
+	buildPipelineServiceAccountName      = "appstudio-pipeline"
+	updateComponentAnnotationName        = "image-controller.appstudio.redhat.com/update-component-image"
+	additionalUsersConfigMapName         = "image-controller-additional-users"
+	additionalUsersConfigMapKey          = "quay.io"
+	SkipRepositoryDeletionAnnotationName = "image-controller.appstudio.redhat.com/skip-repository-deletion"
```

Comment:
```suggestion
	skipRepositoryDeletionAnnotationName = "image-controller.appstudio.redhat.com/skip-repository-deletion"
```
For consistency with the other constants. I think it doesn't need to be exported.

Reply:
done